jni/pubspec.yaml

```diff
@@ -4,7 +4,7 @@
 
 name: jni
 description: A library to access JNI from Dart and Flutter that acts as a support library for package:jnigen.
-version: 0.6.1
+version: 0.7.0-wip
```

Comment:
Could you do a release so that I can regenerate the files for `cronet_http`? We can also do that in a follow-up PR.